src/lib/RouteQueryActionsController.ts

```diff
@@ -38,8 +44,15 @@ export const RouteQueryActionsController = (() => {
     /**
      * Take action after opening the link copied in the transfer receive modal.
      */
-    transferSend: () => {
+    transferSend: (_, query) => {
       const { openModal } = useModals();
+      const { setActiveAccountByProtocol } = useAccounts();
+
+      const token = query[TOKEN_PROP];
+
+      setActiveAccountByProtocol(token === PROTOCOL_BITCOIN
+        ? PROTOCOL_BITCOIN : PROTOCOL_AETERNITY);
```

Comment:
This will break if we add new protocols to the app. I suggest to create come kind of map that would connect the token value prop with the protocols. Or even better set the token name in each of the adapters an loop over them here to find protocol.

Reply:
Sure.